unit_testing/test_reports.py

```diff
@@ -8,31 +8,50 @@
 from spinalcordtoolbox.image import Image
 from spinalcordtoolbox.reports.slice import Sagittal
 
+@pytest.fixture(scope="module")
+def im_base(path_in='t2/t2.nii.gz'):
```

Comment:
Fix for failing CI: (1/2)

```suggestion
def im_base(path_in=sct_test_path('t2', 't2.nii.gz')):
```

Reply:
????

where?

Good catch and fix, thank you, but I found and fixed this bug in #3125. I backed it out of this PR because I thought it only showed up on Actions.

Reply:
Maybe I didn't revert it right.

Reply:
I guess there's no harm in fixing it here instead, though.

Reply:
Ah oops, sorry.

And, my apologies -- these suggestions weren't meant to be committed as-is... `sct_test_path` still needs to be imported. :sweat_smile:

Reply:
Done in https://github.com/neuropoly/spinalcordtoolbox/pull/3152/commits/61933fe6e36f54a82e53732d9943fc1c6ba1cd80.